haiku-apps/xrs/xrs-1.9.1.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="A simple rhythm station for creating music loops"
+DESCRIPTION="XRS is a simple Rhythm Station. You can use it to create music compositions and loops of your preferred elettronic music genres."
+HOMEPAGE="https://github.com/Freaxed/xrs-haiku"
+COPYRIGHT="2006-2022, Andrea Anzani.
+	2017-2020, Andrew Lindesay.
+	2012, Gerasim Troeglazov (3dEyes**).
+	2001-2010 Stephan Aßmus.
+	2006 Javier Serrano Polo.
+	1999 Fabien Fulhaber.
+	1998-1999 Be Incorporated.
+	"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/Freaxed/xrs-haiku/archive/refs/tags/v1.9.1.tar.gz"
+CHECKSUM_SHA256="c2b89a858b4e0ff29104462f7cab80ce318d3c730b559923f5f2477ea687103e"
+SOURCE_DIR="xrs-haiku-1.9.1"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	xrs$secondaryArchSuffix = $portVersion
+	app:XRS$secondaryArchSuffix = $portVersion
```

Comment:
Only one change, for `app` no `secondaryArchSuffix` is needed :)
```diff
- 	app:XRS$secondaryArchSuffix = $portVersion
+ 	app:XRS = $portVersion
```
PS, checked a new checkout, builds fine! Thanks!